web/src/pages/BookWizard/OverviewPage.tsx

```diff
@@ -7,12 +9,108 @@ import '../../App.css';
 type OverviewPageProps = {
   onSubmit: (okLanguages: Map<Language, boolean>) => Promise<void>;
   modalLanguages: Array<Language>
+  title: string
+  author: string
+  image: File | null
+  readTab: TabContent
+  exploreTab: TabContent
+  learnTab: TabContent
+};
+
+type TabPreviewProps = {
+  tab: TabContent
+  type: string
+  title: string
+  author: string
+  image: File | null
+};
+
+function parseYoutubeURL(url: string): string {
+  const regExp = /^.*(youtu\.be\/|v\/|u\/\w\/|embed\/|watch\?v=|&v=)([^#&?]*).*/;
+  const m = url.match(regExp);
+  if (m && m[2].length == 11) {
+    return m[2];
+  }
+  return "";
+}
+
+const TabPreview: React.FC<TabPreviewProps> = ({tab, type, title, author, image}) => {
+  const [imageUrl, setImageUrl] = useState("");
+
+  const mdComponents = {
+    // TODO inc markdown size
```

Comment:
can you do this please? the font for markdown text is really small right now

Reply:
I switched the font size in the css to use em, let me know if that isn't making it any better

Reply:
also this comment can be removed now, thx!